WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2937,6 +2948,59 @@ static GSourceFuncs _handlerIntervention =
             }
         }
 
+        bool GetHDRCapabilities()
+        {
+            int stbCaps = 0;
+            int tvCaps = 0;
+            IARM_Result_t err = IARM_RESULT_SUCCESS;
+            bool retValue = false;
+
+            err = IARM_Bus_Init("wayland-egl-WPEWebProcess");
+            if(IARM_RESULT_SUCCESS != err)
+            {
+                TRACE_L1("Error initializing IARM.. error code : %d\n",err);
+                return retValue;
+            }
+
+            err = IARM_Bus_Connect();
+            if(IARM_RESULT_SUCCESS != err)
+            {
+                TRACE_L1("Error connecting to IARM.. error code : %d\n",err);
+                IARM_Bus_Term();
+                return retValue;
+            }
+
+            device::Manager::Initialize();
+
+            // Get STB HDR capabilities
+            device::VideoDevice decoder = device::Host::getInstance().getVideoDevices().at(0);
+            decoder.getHDRCapabilities(&stbCaps);
+            TRACE_L1("STB HDRCapabilities - [%d]\r\n", stbCaps);
+
+            // Get TV HDR capabilities
+            std::string strVideoPort = device::Host::getInstance().getDefaultVideoPortName();
+            device::VideoOutputPort vPort = device::VideoOutputPortConfig::getInstance().getPort(strVideoPort.c_str());
+
+            if(vPort.isDisplayConnected())
+            {
+                vPort.getTVHDRCapabilities(&tvCaps);
+            }
+
+            TRACE_L1("TV HDRCapabilities - [%d]\r\n", tvCaps);
+
+            if(stbCaps != 0 && tvCaps != 0)
+            {
+                TRACE_L1("STB and TV supports HDR : returning true\r\n");
```

Comment:
I think we don;t need this print as true is returned only in this if statement.

Reply:
Addressed the review comments.